velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2837,3 +2848,86 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
   VELOX_ASSERT_THROW(
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
+
+TEST_F(DateTimeFunctionsTest, dateFunction) {
+  // TIMESTAMP input
+  EXPECT_EQ(std::nullopt, evaluateDateFromTypedInput<Timestamp>(std::nullopt));
+  EXPECT_EQ(
```

Comment:
Please write a lambda for [](Timestamp value)-> void {EXPECT_EQ(evaluateCastDateFromTypedInput(value), evaluateDateFromTypedInput<Timestamp>(value));} and use it for all the tests to make the code more readable.

Same comment for the rest of the tests.

Reply:
I hope it is better now.